content/get-started/index.md

```diff
@@ -0,0 +1,98 @@
+
+If you couldn't find a solution, reach out to the [**Galaxy Help Forum**](https://help.galaxyproject.org/) as *the* place to ask your questions related to Galaxy use or  [**create an issue**](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself. Either way, we are more than happy to help you out!
+
+### Mailing lists
+If you haven't used mailing list before they are simply a list of addresses to which the same information is being sent, just another way to communicate and interact with the Community. See them [**here**](/mailing-lists/) and remember [**don't be afraid to ask questions**](/mailing-lists#new-to-mailing-lists/).
+
+### Galaxy Publication Library
+Check what others members have done in our [**Publication library**](https://www.zotero.org/groups/1732893/galaxy) which contains publications that use, reference, implement, and extend Galaxy.
+
+## Step 3:  Your place within the Galaxy Community
+
+<img class="img-fluid float-right" src="/images/undraw-illustrations/connected-people.svg" style="width:200px;height:100px;" alt="community events" />
+
+Still wonder what the Galaxy Community is about?
+You already know it, you are already part of it! But let’s dive in the complexity of the [Galaxy Communities](/community/)!
+
+### Regional Galaxy Communities
+From India to Australia, Japan to France and many more countries [**Regional Communities**](/community/#regional-communities) are a big part of Galaxy. Find ways to contact them, platforms and lots of other useful links.
+
+### Galaxy Communities of Practice
+
+Another way communities are formed is by domain such as Proteomics, Transcriptomics, Ecology and Genome annotation to name a few. Most have set up their own servers and mailing list, look for them [**here.**](/community/#communities-of-practice)
+
+### Set up your own Community
+What if you can't find yourself in these lists of regional and/or community of practice? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).
+
+## Step 4: Take part and contribute to Galaxy
+
+Look at our [guide](https://galaxyproject.org/community/contributing/) on how to start contributing and there are many ways to do so: coding, writing and creating tutorials, answering questions in Gitter or the Help Forum to mention some ways.
+
+### Working Groups
+Interested in a particular area within Galaxy? Connect with a [**Working Group**](/community/wg/) and look into their Github Project, Gitter channel and Google Drive to look into what they have done, what they're currently working on and what the future holds for each one.
+Some of the current Working Groups include Tools, Systems and User Interface/User Experience.
+
+### Galaxy Mentorship Network
```

Comment:
```suggestion
### Galaxy Mentoring Network
```

Reply:
I think I made this suggestion already? Am I getting confused with the version?

Reply:
I've wrote the same in another file, you're not confused